src/Microsoft.ML.Transforms/WhiteningTransform.cs

```diff
@@ -597,7 +597,7 @@ private static Float DotProduct(Float[] a, int aOffset, Float[] b, int[] indices
 
         private static class Mkl
         {
-            private const string DllName = "Microsoft.ML.MklImports.dll";
+            private const string DllName = "libMklImports";
```

Comment:
This is going to have the same issue as the OLS learner right?  Here is a transform that ships in our `Microsoft.ML` nuget package, that has a dependency on MKL.

However, since this is currently broken (and has been since before v0.1), I am fine with leaving this one broken and logging a separate issue for it.

Reply:
I don't think this is exposed at all at this point. there are more components that are not exposed too. 

---
In reply to: [206640653](https://github.com/dotnet/machinelearning/pull/611#discussion_r206640653) [](ancestors = 206640653)

Reply:
Weird, really?  I didn't know we had transforms that weren't exposed.  Either way, I'm fine with this.